modules/ROOT/pages/log-in-and-log-out.adoc

```diff
@@ -11,13 +11,12 @@ Note: to avoid login problems, make sure you http://www.wikihow.com/Clear-Your-B
 
 == Log in
 
-A user may receive an email with a link to start a case or a do task in Bonita Portal, Bonita User Application, or a custom application.
-Or a user wants to check the status of the tasklist in Bonita Portal, Bonita User Application, or business data and operations in a custom application. +
+A user may receive an email with a link to start a case or do task in Bonita User Application, or a custom application.
```

Comment:
or do task in Bonita User Application, => or do a task in Bonita User Application,

Reply:
Done.